src/types.ts

```diff
@@ -26,3 +26,11 @@ export enum ActionButton {
 }
 
 export type ActionButtonVariant = ActionButton | `${ActionButton}`;
+
+export enum CCSharing {
+  YES = 'yes',
+  NO = 'no',
+  ALIKE = 'alike',
+}
```

Comment:
Should these be part of SDK? And typed into items settings as well? @spaenleh

Reply:
Yes if it goes into `ItemSettings` we should put it in SDK